packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx

```diff
@@ -750,10 +747,17 @@ export const VideoForm: FC<VideoFormProps> = memo(({ onSubmit, setFormStatus })
           <Controller
             control={control}
             name="subtitlesArray"
+            rules={{
+              validate: (value) => {
+                const languageWithoutFile = value?.find((language) => !language.file)
+                return !isEdit && value && languageWithoutFile ? 'All selected languages must include a file!' : true
```

Comment:
Two things here:
1) Why `!isEdit` as a condition for error? I can still add a new subs language and not provide a file when editing
2) Let's change the message to "Provide a file for every new subtitles language."